.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_size = 2
```

Comment:
Missing final blank line. It's quite ironic that it's in `.editorconfig` since you are taking care of it with that file. 🙈